styles/reset.css

```diff
@@ -1,38 +1,109 @@
-
 /* adapted from http://meyerweb.com/eric/tools/css/reset/ 
    v2.0 | 20110126
    License: none (public domain)
 */
 
-*{
+* {
     box-sizing: border-box;
+    border: 1px solid black;
 }
 
-html, body {
+html,
+body {
     height: 100%;
     width: 100%;
+    text-align: center;
+}
+
```

Comment:
If you are wanting the styles here to be global, you instead of listing all of these things you can use the syntax -
`*{
margin: 0;
}` -
if you look at line 6 you can see this has already been applied this way for you.